.cirrus.yml

```diff
@@ -58,7 +58,7 @@ eks_container: &CONTAINER_DEFINITION
   builder_subnet_id: ${CIRRUS_AWS_SUBNET}
   namespace: default
   cpu: 4
-  memory: 4G
+  memory: 6G
```

Comment:
Engine API consumes way less memory than full instantiation of the backend, apparently. After migration of the tests we starting to hit OOM during pipelines run.
Like here for example: https://cirrus-ci.com/task/5742196110917632
Maybe there's a better way to avoid hitting OOM, but increasing memory helps.

Reply:
Added separate commit for memory increase.